src/components/ProjectLineChart.tsx

```
@@ -0,0 +1,328 @@
+    if (hoveredProject) {
+      shownPayload = payload.filter((p: any) => p.name === hoveredProject);
+    }
+
+    const total = shownPayload.reduce(
+      (sum: number, entry: any) => sum + (entry?.value ?? 0),
+      0
+    );
+
+    // Get week start/end dates
+    let weekDates: string | null = null;
+    if (weekMeta && weekMeta[label]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)